### ISSUE TITLE
Unused input

### ISSUE THREAD

**Issue by ma-siddiqui** [NVIDIA-AI-IOT/torch2trt#198]
Hi,
I am facing the below errors while creating cuda engine.

[TensorRT] ERROR: Unused Input: input_0
[TensorRT] ERROR: Unused Input: input_1

**Comment by longnanxi** [comment]
> Hi,
> I am facing the below errors while creating cuda engine.
> 
> [TensorRT] ERROR: Unused Input: input_0
> [TensorRT] ERROR: Unused Input: input_1

Were you able to figure this out?

**Comment by ma-siddiqui** [comment]
Yes. I have upgraded code to latest version of pytorch. Now this error has gone but facing some other errors.

**Comment by czs1886** [comment]
> Yes. I have upgraded code to latest version of pytorch. Now this error has gone but facing some other errors.

Was it caused by the pytorch version?
